scripts/asm-installer/install_asm

```diff
@@ -2751,6 +2755,18 @@ install_managed_components() {
   if [[ "$DISABLE_CANONICAL_SERVICE" -eq 0 ]]; then
     install_canonical_controller
   fi
+
+  info "Configuring ASM managed control plane revision CRD..."
+  kubectl apply -f "${CRD_CONTROL_PLANE_REVISION}"
```

Comment:
Since the CRDs have to be finished processing before the CRs can be applied, I think it makes sense to do this in the "start_managed_control_plane" function where the other base CRDs are applied. (`run kubectl apply --overwrite=true -f "${BASE_REL_PATH}"`)

Reply:
Done